app/models/work-item.ts

```diff
@@ -77,6 +77,20 @@ export default class WorkItem extends Record implements WorkItemRecord {
     const fieldsList = workItems.map(item => _.pick(item, serializedFields));
     await super.insertBatch(transaction, workItems, fieldsList);
   }
+
+  /**
+   * Get the s3 URL to the STAC outputs directory for this work item.
+   * Optionally pass in a target URL in which case the URL returned will be the target URL
+   * resolved relative to the STAC outputs directory.
+   * e.g. s3://artifacts/abc/123/outputs/ with a targetUrl of ./catalog0.json or catalog0.json would resolve to
+   * s3://artifacts/abc/123/outputs/catalog0.json
+   * @param targetUrl - URL to resolve against the base outptuts directory 
+   * @param isAggregate - include the word aggregate in the URL
+   * @returns - the path to the STAC outputs directory (e.g. s3://artifacts/abc/123/outputs/) or the full path to the target URL
+   */
+  getStacLocation(targetUrl = '', isAggregate = false): string {
```

Comment:
I thought it was nice to have it as an instance method as well so that if you have a work item there is one less parameter to pass. But I could get rid of it if you think that's cleaner.